pat-static/src/test/kotlin/com/rectanglel/patstatic/patterns/PatternDataManagerTest.kt

```diff
@@ -144,7 +145,7 @@ class PatternDataManagerTest {
         val file = File("testFiles", filename)
```

Comment:
Since the symlink got deleted, this is now causing this 1 unit test to fail. Possible solutions:

1. (maybe not the best solution) re-add the symlink deleted
2. (maybe better), call a function that returns the testFiles folder which will always make sure to get it at gradle-project-level, then change this code to call that function....

For example, I adding these pieces of code fixed this unit test:

(new kt file in the buildutils package):

```kotlin
package com.rectanglel.patstatic.buildutils

import java.io.File

/**
 * Works around the issue where JUnit from gradlew thinks the relative path is at base project level,
 * while running the Tests in Android Studio thinks the folder is at module-level.
 *
 * This isn't ideal, and I'd rather get a better solution to this...
 */
fun getTestFilesFolder() : File {
    val testFilesFolder = File("testFiles")
    if (testFilesFolder.exists()) {
        return testFilesFolder
    }
    return File("../testFiles")
}
```

Then change this is exact line to this:
```kotlin
val file = File(getTestFilesFolder(), filename)
```